specification/postgresql/Microsoft.OrionDB.Management/examples/2025-08-01-preview/Clusters_CreateOrUpdate.json

```diff
@@ -0,0 +1,203 @@
+            "delegatedSubnetResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.Network/virtualNetworks/test-vnet/subnets/default",
+            "privateDnsZoneResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.Network/privateDnsZones/testcluster1.private.postgres.database.azure.com"
+          },
+          "highAvailability": {
+            "mode": "ZoneRedundant",
+            "standbyAvailabilityZone": "2"
+          },
+          "maintenanceWindow": {
+            "customWindow": "Enabled",
+            "dayOfWeek": 0,
+            "startHour": 2,
+            "startMinute": 0
+          }
+        },
+        "sku": {
+          "name": "Standard_D4s_v3",
+          "tier": "GeneralPurpose"
+        },
+        "identity": {
+          "type": "UserAssigned",
+          "userAssignedIdentities": {
+            "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/test-identity": {
+              "clientId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+              "principalId": "ffffffff-ffff-ffff-ffff-ffffffffffff"
+            }
+          }
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "id": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.OrionDB/clusters/testcluster1",
+        "name": "testcluster1",
+        "type": "Microsoft.OrionDB/clusters",
+        "location": "eastus",
+        "tags": {
+          "Environment": "Development",
+          "Team": "Database"
+        },
+        "properties": {
+          "administratorLogin": "testadmin",
+          "version": "16",
+          "state": "Starting",
+          "availabilityZone": "1",
+          "storage": {
+            "storageSizeGB": 512,
+            "autoGrow": "Enabled",
+            "tier": "P10"
+          },
+          "authConfig": {
+            "activeDirectoryAuth": "Enabled",
+            "passwordAuth": "Enabled",
+            "tenantId": "ffffffff-ffff-ffff-ffff-ffffffffffff"
+          },
+          "dataEncryption": {
+            "type": "SystemManaged"
+          },
+          "backup": {
+            "retentionDays": 7,
+            "geoRedundantBackup": "Disabled"
+          },
+          "network": {
+            "publicNetworkAccess": "Enabled",
+            "delegatedSubnetResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.Network/virtualNetworks/test-vnet/subnets/default",
+            "privateDnsZoneResourceId": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.Network/privateDnsZones/testcluster1.private.postgres.database.azure.com"
+          },
+          "highAvailability": {
+            "mode": "ZoneRedundant",
+            "standbyAvailabilityZone": "2"
```

Comment:
Please sync with Yogesh on this, We can have multiple replicas/Standbys won't give ability to choose AZ for each, rather AZ of primary.

Reply:
Removed standbyAvailabilityZone